test/sanity/validate-modules/validate-modules

```diff
@@ -851,6 +853,30 @@ class ModuleValidator(Validator):
 
         return False
 
+    def _check_for_subprocess(self):
+        for child in self.ast.body:
+            if isinstance(child, ast.Import):
+                if child.names[0].name == 'subprocess':
+                    for line_no, line in enumerate(self.text.splitlines()):
+                        sp_match = SUBPROCESS_REGEX.search(line)
+                        if sp_match:
+                            self.errors.append((
+                                209,
+                                ('subprocess.Popen call found. Should be module.run_command - %d:%d' %
+                                 (line_no + 1, sp_match.span()[0] + 1))
+                            ))
+
+    def _check_for_os_call(self):
+        if 'os.call' in self.text:
+            for line_no, line in enumerate(self.text.splitlines()):
```

Comment:
Here we check if `os.call` exists before looping, and then matching with regex also.  In ` _check_for_subprocess` we don't do this.

As it stands, these 2 new functions don't do things the same way, and they probably should.  That may even allow us to make the function more generic, and only have a single function for matching these types of things.

This could also allows us to consolidate the `sys.exit` check as well.